offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -275,6 +276,16 @@ class CreatePersonView extends VerticalLayout implements Resettable {
         })
     }
 
+    /**
+     * refreshes assuming no organisation is selected
+     */
+    protected void refreshAddressAdditions() {
+        ListDataProvider<Affiliation> dataProvider = new ListDataProvider<>(new ArrayList<Affiliation>())
+        this.addressAdditionComboBox.setDataProvider(dataProvider)
+        this.addressAdditionComboBox.value = addressAdditionComboBox.emptyValue
+        addressAdditionComboBox.setEnabled(false)
+    }
+
```

Comment:
This should already be in development

Reply:
What do you want me to do about it? I think it was because the other hotfix was not merged into dev when I created this PR.

Reply:
I will have a look.